isat/scraper/context.py

```diff
@@ -0,0 +1,24 @@
+import httpx
+import os
+from isat.pkg.logger.logger import Logger
```

Comment:
Not actually bad, but if you execute your app inside pkg, you can get rid of isat.pkg part of import.